docs/source/benchmarking.rst

```diff
@@ -71,73 +90,84 @@ Aggregate across simulated communities (when dataset index is `not` unique)
+        --community-sizes 78Mbp \
+        --file-names binning.tsv.gz\
+        --dir-path simulated
 
-1. Navigate to the 78Mbp community dataset using the `link <https://drive.google.com/drive/u/2/folders/1McxKviIzkPyr8ovj8BG7n_IYk-QfHAgG>`_ mentioned above.
-2. Get the file ID by navigating to any of the files and right clicking, then selecting the ``get link`` option. This will have a ``copy link`` button that you should use. The link for the metagenome assembly (ie. ``metagenome.fna.gz``) should look like this : ``https://drive.google.com/file/d/15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y/view?usp=sharing``
-3. The file ID is within the ``/`` forward slashes between ``file/d/`` and ``/``, e.g:
+Now running the command to benchmark:
 
 .. code:: bash
 
-    # Pasted from copy link button:
-    https://drive.google.com/file/d/15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y/view?usp=sharing
-    #                 begin file ID ^ ------------------------------^ end file ID
+    autometa-benchmark \
+        --benchmark binning-classification \
+        --predictions 78Mbp/binning.tsv \
+        --reference 78Mbp/reference_assignments.tsv.gz \
+        --output-wide binning_classification_wide.tsv.gz
+
+Specifying multiple results
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-4. Copy the file ID
-5. Now that we have the File ID, you can specify the ID or use the ``drive.google.com`` prefix. Both should work.
+``autometa-benchmark`` provides the flexibility of specifying multiple output files of the same dataset and benchmarking them simultaneously. For example, in case you have done the clustering for a sample using three different binner, you can benchmark all of them simulateously. This is really handy when benchmarking multiple tools on the same dataset.
+
+For clustering
+--------------
 
 .. code:: bash
 
-    file_id="15CB8rmQaHTGy7gWtZedfBJkrwr51bb2y"
-    gdown --id ${file_id} -O metagenome.fna.gz
-    # or
-    gdown https://drive.google.com/uc?id=${file_id} -O metagenome.fna.gz
+    autometa-benchmark \
+       --benchmark clustering \
+       --predictions binner1_78Mbp_output.tsv.gz binner2_78Mbp_output.tsv.gz binner3_78Mbp_output.tsv.gz \
+       --reference simulated/78Mbp/reference_assignments.tsv.gz \
+       --output-wide 78Mbp.clustering_benchmarks.wide.tsv.gz \
+       --output-long 78Mbp.clustering_benchmarks.long.tsv.gz
 
-.. note::
+For classification
+------------------
 
-    Unfortunately, at the moment ``gdown`` doesn't support downloading entire directories from Google drive.
-    There is an open `Pull request <https://github.com/wkentaro/gdown/pull/90#issue-569060398>`_ on the ``gdown`` repository
-    addressing this specific issue which we are keeping a close eye on and will update this documentation when it is merged.
+.. code:: bash
 
+    autometa-benchmark \
+        --benchmark classification \
+        --predictions taxonomic_profiler1_78Mbp.tsv.gz taxonomic_profiler2_78Mbp.tsv.gz taxonomic_profiler3_78Mbp.tsv.gz  \
```

Comment:
Why not list some of the taxon-profilers outputs in KwanLab/metaBenchmarks?

Reply:
See comment above